chore: rename organization and project

Pangssu opened the pull request:
As part of the rebranding of WEMIX Kanvas to **Kroma**, this commit includes changes to rename the name of the GitHub organization and the project to **Kroma**:

- All components in the format of kanvas-xxx are changed to kroma-xxx.
- The organization name, previously wemixkanvas, is changed to kroma-network.
- The wemixkanvas.io domain is changed to kroma.network.
- All other texts related to Kanvas are changed to Kroma.

chokobole commented on specs/differences-from-optimism-bedrock.md:
```diff
@@ -25,18 +25,18 @@
 | Node        | Components                                                                          |
 |-------------|-------------------------------------------------------------------------------------|
-| `proposer`  | `L2 EL client` + `L2 CL client` + `kanvas-batcher`                                  |
-| `validator` | `L2 EL client` + `L2 CL client` + `kanvas-validator` + (optionally `kanvas-prover`) |
```
Please align the table cells

chokobole commented:
FYI, I can see `Optimism` keyword under `bindings/hardhat/testdata/deployments`.

Pangssu commented:
> FYI, I can see `Optimism` keyword under `bindings/hardhat/testdata/deployments`.

removed legacy testdata and added sepolia and easel deployments to testdata

seolaoh commented:
> npm organization `@kroma` is available, so please replace `@kroma-network` with `@kroma`.

LGTM except the name of npm org mentioned above.

chokobole commented:
I added little fixes on top of your change. Please review the last change @Pangssu @seolaoh.

seolaoh reviewed (approved):
LGTM

Pangssu commented:
Squashed
